woocommerce/woocommerce-gateway-stripe
# Update the importing and alt property of payment method icons

## Changes proposed in this Pull Request:

- Add an "alt" property for the payment method icon images with their name as the value.
- Include Alipay icon on the index file of the payment method icons
- Update how payment method icons are imported for the payment method map


## Testing instructions

1. Visit the following pages:
   - Block checkout page
   - Stripe settings page > Payment methods tab, at `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=methods`
  
2. Confirm that:
- The payment method icons are displayed without errors
- The `img` tag for the icons have an `alt` property with the payment method label


**Test Alipay icon is displayed on the Block checkout page**
1. Set USD as the store currency
2. Disable the Legacy experience, if it's enabled. At `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings`
3. Enable Alipay. At `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=methods`
4. Go to the Block checkout page
5. Confirm that the Alipay icon is displayed next to the Alipay payment method option

---


**Post merge**
